internal/views/pod.go

```diff
@@ -51,10 +51,12 @@ func newPodView(title, gvr string, app *appView, list resource.List) resourceVie
 
 func (v *podView) extraActions(aa ui.KeyActions) {
 	aa[tcell.KeyCtrlK] = ui.NewKeyAction("Kill", v.killCmd, true)
-	aa[ui.KeyS] = ui.NewKeyAction("Shell", v.shellCmd, true)
+	if !v.list.HasMarks() { // FIXME: actions list is not refreshed with resourceView.refresh()
```

Comment:
Seems like header is never updated. This line is hit every "refresh" but the removed actions are still present